OpenDF-Describe-Image/src/Server/server.py

```diff
@@ -23,16 +22,19 @@
 app = Flask(__name__)
 app.config['UPLOAD_FOLDER'] = UPLOAD_FOLDER
 
+#images dictionary contains uuids and corresponding file path for each uuid
+images = {}
+
 @app.route('/uploads/<filename>')
 def uploaded_file(filename):
     return send_from_directory(app.config['UPLOAD_FOLDER'],
                                filename)
+#images dictionary must have at least one record
+def get_actual_path(uuid):
```

Comment:
This will be a temporary function. Once completed, this function will fetch these details from Elastisearch.